### ISSUE TITLE
Ne fonctionne plus avec la nouvelle version ?

### ISSUE THREAD

**Issue by OverSu** [alexAubin/yunoScripts#1]
A priori ce script - que j'utilise depuis 2 ans - ne fonctionne plus sur la nouvelle version ?

Une idée de comment récupérer les DKIM et SPF nécessaire ? 🗡

**Comment by alexAubin** [comment]
Est-ce que tu peux expliquer pourquoi il ne fonctionne plus ? (message d'erreur ...?)

Tu peux avoir ta conf DNS recommandée via "yunohost domain dns-config" (ou bien dans la webadmin...)

**Comment by OverSu** [comment]
Alors, je n'ai aucun message d'erreur, le script ne s'arrête pas. Il attend quelque chose ?

J'ai testé ton autre script yunoDKIM, qui lui fonctionne bien.

La commande renvoi: yunohost domain dns-conf: error: too few arguments
(J'ai testé yunohost domain dns-config ou yunohost domain dns-conf)

:D

**Comment by alexAubin** [comment]
Uuuuuh, mais tu parles de quel script alors ... @_@

**Comment by OverSu** [comment]
Je te parle du script yunoCheckDNS. Celui pour checker les DKIM et SPF. ^^ '

**Comment by alexAubin** [comment]
Mouaip du coup la commande dont je parlais c'est `yunohost domain dns-conf` suivi du domaine que tu veux..

**Comment by OverSu** [comment]
J'ai ça :

> Info: This command shows you the *recommended* configuration. It does not actually set up the DNS configuration for you. It is your responsability to configure your DNS zone in your registrar according to this recommendation.
> ; Basic ipv4/ipv6 records
> @ 3600 IN A 163.172.210.71
> * 3600 IN A 163.172.210.71
> 
> ; XMPP
> _xmpp-client._tcp 3600 IN SRV 0 5 5222 oversu.fr.
> _xmpp-server._tcp 3600 IN SRV 0 5 5269 oversu.fr.
> muc 3600 IN CNAME @
> pubsub 3600 IN CNAME @
> vjud 3600 IN CNAME @
> 
> ; Mail
> @ 3600 IN MX 10 oversu.fr.
> @ 3600 IN TXT "v=spf1 a mx ip4:163.172.210.71 -all"
> mail._domainkey 3600 IN TXT "v=DKIM1; h=sha256; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDIPW58EBePPBHrVAmv+NX/Wljo/wrd18SjovlexrrCif5wl94MQTHnH9JBvkd4JuJGJef1Pke2H9wRYak2kNq7xwHXUZwsNuuwa2TYLFqZ6yptjYLgJA0lAbbPCmKJ0V+DjITYvE9z30PLcZyANFjCkOu4VmIoRLo3VSozPemPEwIDAQAB"
> _dmarc 3600 IN TXT "v=DMARC1; p=none"
> 
> ; Extra
> @ 3600 IN CAA 128 issue "letsencrypt.org"
> `

**Comment by alexAubin** [comment]
Oui, du coup ça réponds a ta question "comment récupérer les DKIM et SPF nécessaire ?" ...

**Comment by OverSu** [comment]
... Oui °°
